x/supply/keeper/bank.go

```diff
@@ -0,0 +1,138 @@
+package keeper
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/supply/types"
+)
+
+// SendCoinsFromModuleToAccount trasfers coins from a ModuleAccount to an AccAddress
```

Comment:
```suggestion
// SendCoinsFromModuleToAccount transfers coins from a ModuleAccount to an AccAddress
```
misspelled throughout this file

Reply:
good catch ! thanks